boss-ws/src/main/java/org/genomebridge/boss/http/models/ResolveRequest.java

```diff
@@ -0,0 +1,19 @@
+package org.genomebridge.boss.http.models;
+
+
+import com.wordnik.swagger.annotations.ApiModel;
+import com.wordnik.swagger.annotations.ApiModelProperty;
+
+@ApiModel("Resolve Request")
+public class ResolveRequest {
+
+    @ApiModelProperty(value = "value must be strictly positive",dataType = "Integer")
```

Comment:
Language: "The number of seconds that this request will be valid for. This value must be positive."